binderhub/launcher.py

```diff
@@ -30,15 +30,47 @@ class Launcher(LoggingConfigurable):
 
     hub_api_token = Unicode(help="The API token for the Hub")
     hub_url = Unicode(help="The URL of the Hub")
+    retries = Integer(
+        4,
+        config=True,
+        help="""Number of attempts to make on Hub API requests.
+
+        Adds resiliency to intermittent Hub failures,
+        most commonly due to Hub, proxy, or ingress interruptions.
+        """
+    )
+    retry_delay = Integer(
+        10,
```

Comment:
10 seems like quite a lot, no? That means that by the 4th delay it will be waiting for 40 seconds before proceeding right?

Reply:
switched to exponential backoff with an initial time of 4 seconds, so: 4, 8, 16, 32.